pkg/trustedresources/verify.go

```diff
@@ -39,12 +42,24 @@ const (
 )
 
 // VerifyTask verifies the signature and public key against task.
+// Skip the verification when no policies are found and trusted-resources-verification-no-match-policy is set to ignore or warn
```

Comment:
```suggestion
// Skip the verification when no policies are found and trusted-resources-verification-no-match-policy is set to ignore or warn
// Returns an error when no policies are found and trusted-resources-verification-no-match-policy is set to fail
```

Reply:
Thanks! I added all the error cases in the docs